### ISSUE TITLE
[P3.2][Sourcing][Accessibility][Dialog] Reopening of 10698

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#11377]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
This issue is created as https://github.com/SAP/fundamental-ngx/issues/10698 was not being addressed for dialogs such as fd-dialog and value help dialog are reading chrome tab name

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core/platform: 0.43.26

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Fd-dialog and value help dialog pls refer https://github.com/SAP/fundamental-ngx/issues/10698 for more info

**Comment by droshev** [comment]
@nikolay-kolarov We would need some guidance from you. :)

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

In the current example the value help dialog does not have an aria-labelledby attribute pointing to the title and I think that if it this is added, then the announcement of the page title will be gone. Could you please try this?

Thanks,
Nikolay

**Comment by sagar7259** [comment]
Hello,

The issue with reading chrome tab name is only fixed for value-help dialog components and not for fd-dialogs.
Here in fd-dialog, this issue still persists, where the chrome tab title is still being read.

https://deploy-preview-11651--fundamental-ngx.netlify.app/#/core/dialog
![Screenshot (1693)](https://github.com/SAP/fundamental-ngx/assets/143692208/68c7acaf-6118-49cd-8413-8c6609099feb)

Kindly reopen the ticket.

Regards,
Sagar

**Comment by droshev** [comment]
@sagar7259 Can you create a stackblitz example only with the dialog?

**Comment by mikerodonnell89** [comment]
I am seeing the problem on the provided stackblitz, and the deploy preview from the relevant PR (#11651), but not on the latest deployment of the ng-15 downport:

<img width="1223" alt="Screenshot 2024-04-29 at 4 05 02 PM" src="https://github.com/SAP/fundamental-ngx/assets/2471874/b08110bf-8c44-4098-9f09-24b9b6b1a8f6">
